Automattic/woocommerce-payments
# Add support for UAE

Fixes #6480 

#### Changes proposed in this Pull Request

Add fee support for United Arab Emirates









#### Testing instructions
* On server side checkout 3557-gh-Automattic/woocommerce-payments-server
* **Base fees**
* Go to Woo -> Settings -> General and set Store Country to United Arab Emirates and currency to AED
* Place 2 orders and pay it with an AE credit card and international credit card
* check the applied fees on the order transaction ( in admin, Payments->Transactions )






*

-------------------


**Post merge**
